native/src/boot/bootimg.cpp

```diff
@@ -479,14 +494,23 @@ int unpack(const char *image, bool skip_decomp, bool hdr) {
     dump(boot.kernel_dtb, boot.hdr->kernel_dt_size, KER_DTB_FILE);
 
     // Dump ramdisk
-    if (!skip_decomp && COMPRESSED(boot.r_fmt)) {
-        if (boot.hdr->ramdisk_size() != 0) {
-            int fd = creat(RAMDISK_FILE, 0644);
-            decompress(boot.r_fmt, fd, boot.ramdisk, boot.hdr->ramdisk_size());
+    for (const auto& [entry, fmt]: boot.ramdisk_table_entries) {
+        // change file name only for vendor_boot v4 to keep backwards compatibility with scripts etc.
```

Comment:
Magisk has never used vendor_boot, does this comment make sense?

Reply:
Why would it need a --vendor switch when it already uses a different header magic? Shouldn't the installer script just reject VNDR magic?

Reply:
This is actually for boot image.